### PR TITLE
improvement: update formatting in docs

### PR DESCRIPTION
**Pull request by rahulreddy**

### CODE REVIEW

File: docs/GETTING_STARTED.rst [hunk 1]
```diff
@@ -1,122 +1,100 @@
...
 
-Clone source code
-~~~~~~~~~~~~~~~~~
+
+Clone the Source Code
+~~~~~~~~~~~~~~~~~~~~~
 
 .. code:: shell
 
     git clone https://github.com/scality/S3.git
```
Comment (bennettbuchanan) [inline docs/GETTING_STARTED.rst:27]: Do we also want to update references to the S3 repo name to cloudserver?

File: docs/GETTING_STARTED.rst [hunk 2]
```diff
@@ -1,122 +1,100 @@
...
 
-Install js dependencies
+Install js Dependencies
```
Comment (bennettbuchanan) [inline docs/GETTING_STARTED.rst:29]: ~js~ JS

**Review by bennettbuchanan — APPROVED** [review]

File: docs/GETTING_STARTED.rst (hunk 1 above)
Comment (philipyoo) [inline docs/GETTING_STARTED.rst:27]: cloudserver

File: docs/GETTING_STARTED.rst [hunk 3]
```diff
@@ -1,122 +1,100 @@
...
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Go to the ./S3 folder,
+Go to the ./S3 folder and enter:
```
Comment (philipyoo) [inline docs/GETTING_STARTED.rst:32]: `./cloudserver`

File: docs/GETTING_STARTED.rst [hunk 4]
```diff
@@ -1,122 +1,100 @@
...
+The default access key is accessKey1, with a secret key of verySecretKey1.
 
-By default the metadata files will be saved in the localMetadata
-directory and the data files will be saved in the localData directory
-within the ./S3 directory on your machine. These directories have been
-pre-created within the repository. If you would like to save the data or
-metadata in different locations of your choice, you must specify them
-with absolute paths. So, when starting the server:
+By default, metadata files are saved in the localMetadata directory and data
+files are saved in the localData directory in your machine's ./S3 directory.
```
Comment (philipyoo) [inline docs/GETTING_STARTED.rst:53]: ./cloudserver

File: docs/GETTING_STARTED.rst [hunk 5]
```diff
@@ -1,122 +1,100 @@
...
-accessKey1 with a secret key of verySecretKey1.
+starts a Zenko CloudServer on port 8000. The default access key is
+accessKey1, with a secret key of verySecretKey1.
 
-Run it for continuous integration testing or in production with Docker
-----------------------------------------------------------------------
+Run CloudServer for Continuous Integration Testing or in Production with Docker
+-------------------------------------------------------------------------------
 
 `DOCKER <../DOCKER/>`__
```
Comment (philipyoo) [inline docs/GETTING_STARTED.rst:81]: `../DOCKER.rst`

File: docs/GETTING_STARTED.rst [hunk 6]
```diff
@@ -126,39 +104,40 @@ You can run the linter with:
...
 
-::
+1.  Install `s3cmd <http://s3tools.org/download>`__
+2.  Install `redis <https://redis.io/download>`__ and start Redis.
+3.  Add localCache section to your ``config.json``:
+
+  .. code:: json
 
     "localCache": {
         "host": REDIS_HOST,
```
Comment (philipyoo) [inline docs/GETTING_STARTED.rst:124]: same red background here

**Review by jonathan-gramain — APPROVED** [review]

**Review by smaffulli — APPROVED** [review]
LGTM

**Pull request closed without merging**